tests/lean/run/coe_to_fn.lean

```diff
@@ -21,6 +21,6 @@ local postfix `⁻¹` := equiv.inv
 def equiv.trans (f : α ≃ β) (g : β ≃ γ) : α ≃ γ :=
 ⟨g ∘ f, f⁻¹ ∘ g⁻¹⟩
 
-example (f : α ≃ β) := function.bijective f
+example (f : α ≃ β) := @function.bijective α β f
```

Comment:
Issue 6: Why do I need to fill these in explicitly? If I don't I get an error that seems to be related to the lack of reduction of `domain` and `codomain`